### ISSUE TITLE
HTTP Write Error: Request Entity Too Large (Missing feature Customization)

### ISSUE THREAD

**Issue by sandebains** [influxdata/influxdb#9214]
### Directions
_GitHub Issues are reserved for actionable bug reports and feature requests._
_General questions should be sent to the [InfluxDB Community Site](https://community.influxdata.com)._

_Before opening an issue, search for similar bug reports or feature requests on GitHub Issues._
_If no similar issue can be found, fill out either the "Bug Report" or the "Feature Request" section below.
_Erase the other section and everything on and above this line._

### Bug report

__System info:__ [1.4.2, CENTOS, and Indexing: TSi1]

__Steps to reproduce:__

1. Using Go client for Influxdb, write the data to Influxdb. Since the upgrade for certain files I'm seeing the error "Request Entity Too Large"

Based on the document listed under the following link: "https://github.com/influxdata/docs.influxdata.com/blob/master/content/influxdb/v1.3/administration/config.md
This is a configurable option. I modified the influxdb.conf with the following 
max-body-size = 0

After the change the database influxdb process was restarted. But it hasn't made any difference.

__Expected behavior:__ [The setting should help with the issue]

__Actual behavior:__ [It seems to be hard coded]

**Comment by kmova** [comment]
I am hitting the same error with influx version 1.4.2

**Comment by itsarbit** [comment]
I am hitting the same error too. The version is also 1.4.2

**Comment by lukaszgryglicki** [comment]
Same with me on v1.5.1

**Comment by lukaszgryglicki** [comment]
Sorry, seems like after restarting `influxd` it actually worked.
I had a long running process on the other server, where I didn't restart `influxd`, so v1.5.1 is OK for me in thi case.

**Comment by krbvroc1** [comment]
I am using the official influx v2 go client. I create a NewHTTPClient and a NewBatchPoints. Every 30 seconds I add some point and write them. This runs in an infinite loop. After some period of time (days) I see this error:
```
[snip...running for a week...]
May 24 02:23:24 influxd[831]: [httpd] ::1 - stats [24/May/2018:02:23:21 -0400] "POST /write?consistency=&db=ups&precision=s&rp= HTTP/1.1" 204 0 "-" "InfluxDBClien
[and from here on it fails with 413 - Request Entity Too Large]
May 24 02:23:57 influxd[831]: [httpd] ::1 - stats [24/May/2018:02:23:57 -0400] "POST /write?consistency=&db=ups&precision=s&rp= HTTP/1.1" 413 37 "-" "InfluxDBClie
```
For me simply restarting the go client works. Is there some memory leak or something in influxdb?

**Comment by lukaszgryglicki** [comment]
I've seen something similar.... after some days DB processes were using more and more memory, sometimes generating errors.
I've always restarted `influxd` and it seem to help for some time.
But I cannot say for sure, never did such a specific test. And don't remember now which influx version it was. I've used many of them.

We replaced Influx with Postgres and now all is working OK and faster.
